src/main/java/duke/DukeTaskList.java

```diff
@@ -0,0 +1,75 @@
+package duke;
+
+import task.Task;
+
+import java.util.ArrayList;
+
+/**
+ * Handles duke task list operations.
+ */
+public class DukeTaskList {
+    /**
+     * Contains the currently stored tasks.
+     */
+    protected ArrayList<Task> taskList;
+
+    /**
+     * Instantiates a new Duke task list.
+     *
+     * @param taskList the task list
+     */
+    DukeTaskList(ArrayList<Task> taskList) {
+        this.taskList = taskList;
+    }
+
+    /**
+     * Add new task into task list.
+     *
+     * @param t the t
```

Comment:
The description for the @param could be clearer. I've noticed the same issue in some other places as well.